pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -657,10 +955,19 @@ func (m *kubeGenericRuntimeManager) computePodActions(pod *v1.Pod, podStatus *ku
 		var reason containerKillReason
 		restart := shouldRestartOnFailure(pod)
 		if _, _, changed := containerChanged(&container, containerStatus); changed {
```

Comment:
I think of a situation, whether the container changes is judged according to the container hash. For example, if the cpu is expanded from 2 to 3, the resize can be successful, and when it is retracted from 3 to 2, the container hash does not change, so it will not enter the computePodResizeAction function, the scaling will fail at this time.

Reply:
Ok, I will test the case on e2e later, resource A resize to resource B, and then resource B resize to resource A, to see if it can work normally in this mode.